server/monitor/src/main/resources/org/apache/accumulo/monitor/templates/scans.ftl

```diff
@@ -17,6 +17,11 @@
       <div><h3>${title}</h3></div>
       <div class="center-block">
         <table id="scanStatus" class="table table-bordered table-striped table-condensed">
-
+            <caption><span class="table-caption">Scan&nbsp;Status</span><br/></caption>
+            <tbody><tr><th class="firstcell" onclick="sortTable(0)">Server&nbsp;</th>
+                <th onclick="sortTable(1)" title="Number of scans presently running">#&nbsp;</th>
```

Comment:
Might be able to change `sortTable` implementation to automatically detect which column number to sort on. May have to pass `sortTable(this)`, but probably not. This isn't important, but just an idea for an improvement, since the numbers could result in copy/paste errors if the columns are reordered or new columns are added and the parameters aren't updated. Making the method call resilient in the face of column reordering or adding, would be nice in the future.

Reply:
Created a newbie ticket as I thought this would be a good ticket for newcomers since it only requires knowledge of javascript to complete.